CoreAlgorithmPlugins/release/modules/ext/docs/CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/hierarchical-clustering.md

```diff
@@ -36,9 +36,28 @@ panel from Tools -> Cluster -> Hierarchical. To apply the changes from the clust
 to the graph click the Cluster button and then Toggle Interactive: Disabled. This
 will change the color of the graph elements to match the clusters. 
 
-The arrows on the right side allow for the number of clusters to be modified and the 
-<img src="../ext/docs/CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/resources/shortestpaths.png" width="16" height="16" />
-button will indicate the Shortest Path between cluster centres. 
+<br />
+<div style="text-align: center">
+<img src="../ext/docs/CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/resources/clusterPanel.png" width="1000" />
+</div>
+<br />
+
+### Adjusting Number of Clusters
+
+The arrows on the right side move the marker along the clustering display, 
+allowing for the number of clusters to be chosen. 
+The optimal clustering button will adjust the number of clusters such that clustering is optimal.
+
+### Shortest Path
+
+The <img src="../ext/docs/CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/resources/shortestpaths.png" width="16" height="16" />
+button will select the shortest path between the centers of all clusters
+
+### Excluding Elements
+
+Elements that are not part of clustering, such as unnecessary nodes or transactions, can have their visibilty altered.
+They can be either showm, dimmed or hidden with the respective radio buttons located on the left of the Hierarchical Clustering panel.
```

Comment:
"shown". Also should "Hierarchical Clustering panel" become "Hierarchical Clusters panel" since that is the name displayed in the application?